driver_macos.go

```diff
@@ -17,7 +17,50 @@
 
 package oto
 
-// #cgo LDFLAGS: -framework AppKit
-import "C"
+import (
+	"unsafe"
+
+	"github.com/ebitengine/purego"
+	"github.com/ebitengine/purego/objc"
+)
 
 const bufferSizeInBytes = 2048
+
+var appkit = purego.Dlopen("/System/Library/Frameworks/AppKit.framework/Versions/Current/AppKit", purego.RTLD_GLOBAL)
+
+// oto_setNotificationHandler sets a handler for sleep/wake notifications.
+func oto_setNotificationHandler() {
+	// Create the Observer object
+	class := objc.AllocateClassPair(objc.GetClass("NSObject\x00"), "OtoNotificationObserver\x00", 0)
+	class.AddMethod(objc.RegisterName("receiveSleepNote:\x00"), objc.IMP(oto_setGlobalPause), "v@:@\x00")
+	class.AddMethod(objc.RegisterName("receiveWakeNote:\x00"), objc.IMP(oto_setGlobalResume), "v@:@\x00")
+	class.Register()
+	//OtoNotificationObserver *observer = [OtoNotificationObserver new];
```

Comment:
Is it intended to leave comments for the original ObjC code?

Reply:
It was but I don't think it's needed if you'd rather have them removed

Reply:
Thanks for removing them!